### PR TITLE
Updated key shortcuts to latest bindings from Design

### PR DESCRIPTION
**Pull request by steveyegge**
Updated the JetBrains keyboard shortcuts according to the instructions [here](https://linear.app/sourcegraph/document/keyboard-shortcuts-for-cody-on-jetbrains-2f8747a22530).

## Test plan

I wrote a unit test for the bindings. I would have liked to trigger the keystrokes programmatically in the test (i.e., an integration test), and then listened for the resulting actions to fire. However, without the UI-robot tests, I was not able to find a way to simulate a keypress to trigger an action.

So instead, I wrote a simpler and slightly weaker test, which looks up all the actions of interest, gets their registered key bindings, and compares them against the expected test values from Design. This unfortunately doesn't exercise all the code paths -- specifically it isn't really testing the routing for retry and for the undo/cancel/dismiss cluster, and it's not guaranteeing that the actions will trigger even if the bindings are registered. But it will definitely catch whether we have the wrong bindings for the top-level actions.

I have not manually tested this on Windows yet, because I still need to set up Parallels, and I'm headed out for PTO. But the unit test should work on Windows as well.

### CODE REVIEW

**Review by mkondratek — APPROVED** [review]
I did not test it myself but it looks good 👍 

pls make sure it works as expected on macOS and Windows

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -163,13 +169,13 @@
         </action>
         <action id="cody.cycleForwardAutocompleteAction"
                 class="com.sourcegraph.cody.autocomplete.action.CycleForwardAutocompleteAction">
-            <keyboard-shortcut first-keystroke="alt OPEN_BRACKET"
-                               keymap="$default"/>
+            <keyboard-shortcut first-keystroke="alt OPEN_BRACKET" keymap="$default"/>
```
Comment (pkukielka) [inline src/main/resources/META-INF/plugin.xml]: Minor thing, but I think we should add `replace-all="true"` there as in other places. 
That ensures that if we ever change the binding in the plugin configuration old one will be removed.

File: src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          action.shortcutSet.shortcuts
+              .mapNotNull { shortcut ->
+                when (shortcut) {
+                  is KeyboardShortcut -> shortcut.firstKeyStroke
+                  else -> null
+                }
+              }
+              .toTypedArray()
+
+      val key = if (SystemInfoRt.isMac) macKey else windowsKey
```
Comment (pkukielka) [inline src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt:61]: what is expected behaviour on the Linux CI?

**Review by pkukielka — APPROVED** [review]
LGTM, I added minor comments.
Also - PR requires spotless reformatting.

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (steveyegge) [reply src/main/resources/META-INF/plugin.xml]: Fixed!

File: src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt (hunk 2 above)
Comment (steveyegge) [reply src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt:61]: cc: @danielmarquespt -- I think we need to formally define the Linux bindings.

**Pull request merged**